shell/gpu/gpu_surface_gl_delegate.h

```diff
@@ -60,6 +62,9 @@ class GPUSurfaceGLDelegate : public GPUSurfaceDelegate {
   // instrumentation to specific GL calls can specify custom GL functions
   // here.
   virtual GLProcResolver GetGLProcResolver() const;
+
+  // |GPUSurfaceDelegate|
+  ExternalViewEmbedder* GetExternalViewEmbedder() override;
```

Comment:
Minor nit:  Can you move this to the top, below the destructor?  Its more visible that way and the class flows better